crates/sparrow-compiler/src/dfg.rs

```diff
@@ -108,6 +108,7 @@ impl Default for Dfg {
 impl Dfg {
     pub(super) fn add_literal(&mut self, literal: impl Into<ScalarValue>) -> anyhow::Result<Id> {
         let literal = literal.into();
+        // TODO: FRAZ - do I need to support large string literal here?
```

Comment:
Maybe. We're going to potentially run into many problems with things (eg., what is the type of the literal `"hello"`?) This is the same as numbers (where we previously assumed it was `i64`).

Reply:
I'm not sure.. I think if we assume that every string literal is just a `string`, then we can rely on the `lub` of `string` and `large_string` to cast to `large_string` if necessary?